.travis.yml

```diff
@@ -20,3 +20,40 @@ script:
   - \[ "$(sed -nE 's/^molo\.core==(.*)$/\1/p' requirements.txt)" = "$(sed -nE 's/^FROM .*:(.*)-onbuild$/\1/p' Dockerfile)" \]
 after_success:
   - coveralls
+
+
+jobs:
+  include:
+    - stage: test
+    - stage: docker
+      python: "2.7"
+      sudo: required
+      dist: trusty
+      services: [docker]
+      env:
+        - IMAGE_NAME=praekeltfoundation/molo-gem
+        - REGISTRY_USER=praekeltorgdeploy
+        - secure: ""
+
+      # Update docker-engine: we need at least 17.03.1 for `--cache-from`
```

Comment:
This comment is not strictly correct anymore but we probably still want to do this. Maybe make it more vague :)
```
# Update Docker: we want some new docker build features
```